pegasus/routes/poste_routes.rb

```diff
@@ -63,8 +63,21 @@
   template_params = JSON.parse(params[:params]) unless params[:params].to_s.empty?
   template_params ||= {}
 
+  recipients_csv = CSV.parse(params[:recipients_file][:tempfile].read, {headers: true})
+
   recipients = params[:recipients].to_s.split(/[\n,;]/).map(&:strip)
 
+  recipients_csv.each do |recipient|
+    recipients << recipient["email"]
+  end
+
+  if params[:template_file]
+    DB[:poste_messages].insert(
+      name: params[:template_file][:filename],
+      template_file: params[:template_file][:tempfile].read
```

Comment:
or do you want to leave this change out for now since you haven't added a field to the UI for it?